tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/javainterop/basic/NegativeValidationTest.java

```diff
@@ -83,42 +82,39 @@ public void testMethodNotFound2() {
     public void testMethodNotFound3() {
 
         String path = "test-src/javainterop/negative/method_not_found3.bal";
-
         CompileResult compileResult = BCompileUtil.compile(path);
         compileResult.getDiagnostics();
         Assert.assertEquals(compileResult.getDiagnostics().length, 1);
         BAssertUtil.validateError(compileResult, 0,
                 "{ballerina/jballerina.java}METHOD_NOT_FOUND 'No such public static method " +
                         "'acceptRecordAndRecordReturn' with '3' " + "parameter(s) found in class " +
-                        "'class org.ballerinalang.nativeimpl.jvm.tests.StaticMethods''",
+                        "'org.ballerinalang.nativeimpl.jvm.tests.StaticMethods''",
                 "method_not_found3.bal", 21, 1);
     }
 
     @Test
     public void testMethodNotFound4() {
 
```

Comment:
```suggestion
```

Reply:
Are we following any coding convention to have a new line between the method signature and method body?

Reply:
I was told that previously there was such convention and that had been added in the checkstyle.xml file also. But that was removed and now we are following not to keep newlines between the method signature and method body.